src/Sylius/Bundle/ApiBundle/spec/Serializer/ProductImageNormalizerSpec.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace spec\Sylius\Bundle\ApiBundle\Serializer;
+
+use PhpSpec\ObjectBehavior;
+use Sylius\Component\Core\Model\OrderInterface;
+use Sylius\Component\Core\Model\ProductImageInterface;
+use Symfony\Component\Serializer\Normalizer\ContextAwareNormalizerInterface;
+use Symfony\Component\Serializer\Normalizer\NormalizerInterface;
+
+final class ProductImageNormalizerSpec extends ObjectBehavior
+{
+    function let(): void
+    {
+        $this->beConstructedWith('prefix');
+    }
```

Comment:
Constructor has 4 different paths that can be executed, only the first if is tested. We should cover 3 additional cases:
* `/prefix`
* `/prefix/`
* `prefix/`